Swolify/swolify-fe
# Feature/demo landing

# <p align="center">Swolify PR</p>
<hr>

## What will this PR do?
<hr>

- Adds a basic profile selection page that links to a currently un-built user dashboard


## Where should the reviewer start?
<hr>

- ProfileView.js


## How should this be manually tested?
<hr>

- Clone down the repo, cd into it, run "npm start". A user should see a landing page of 4 user icons, and upon clicking one, should be taken to a link that reads "localhost:3000/dashboard"


## Any background context you want to provide?
<hr>

- no


## What are the relevant issues?
<hr>

- closes #12 


## Screenshots (if appropriate)
<hr>

-



## Pull request checklist:
<hr>

| Publisher: | Reviewer: |
| ---------- | --------- |
|<ul><li>- [x] I tested this locally for bugs and errors before pushing.</li><li>- [x] This code breaks no existing tests.</li><li>- [x] I have conducted a refactor on the code to remove comments and consoles.</li><li>- [x] I have requested someone to review my PR.</li></ul> | <ul><li>[ ] I have run test files to confirm this does not fail any existing tests.</li><li>[ ] I have reviewed all files edited</li><li>- [ ] I have asked for clarifying information when I am unclear</li><li>- [ ] This follows best practices and team decided norms.</li></ul>| 


## Questions:
<hr>

-
